### PR TITLE
12.0 polling

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_longpolling/static/src/js/pos.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+odoo.define('pos_longpolling.pos', function(require){
+    "use strict";
+
+    var models = require('point_of_sale.models');
+    var PosLongpolling = require('pos_longpolling.connection');
```
Comment (yelizariev) [inline pos_longpolling/static/src/js/pos.js]: Please name the class equal to one in pos_longpolling.connection'

File: pos_longpolling/static/src/js/pos.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+odoo.define('pos_longpolling.pos', function(require){
+    "use strict";
+
+    var models = require('point_of_sale.models');
+    var PosLongpollingConnection = require('pos_longpolling.connection');
```
Comment (yelizariev) [inline pos_longpolling/static/src/js/pos.js]: It's PosConnection in ``'pos_longpolling.connection``

**Review by yelizariev — APPROVED** [review]

**Review by KolushovAlexandr — APPROVED** [review]
The code was totally refactored, almost all relations and chains of functions are different now. It works. It would be better to test it together with the ported pos_multi_session_sync module

**Comment by GabbasovDinar** [comment]
@KolushovAlexandr You can test here https://github.com/it-projects-llc/pos-addons/pull/790

**Comment by yelizariev** [comment]
@KolushovAlexandr I still need your review for the code itself

**Comment by yelizariev** [comment]
At least about refactored part

**Review by Ramil-Mukhametzyanov — APPROVED** [review]
It works. I checked it on the Discuss app.

**Review by KolushovAlexandr — APPROVED** [review]
I checked changes locally, they work correctly

**Pull request merged**